### ISSUE TITLE
Add Informational message to indicate no results are found in case of no match

### ISSUE THREAD

**Issue by bauthard** [projectdiscovery/nuclei#96]
Related issue https://github.com/projectdiscovery/nuclei/issues/95, https://github.com/projectdiscovery/nuclei/issues/94

**Comment by m0nk3y-s3c** [comment]
Just to be sure there is no issue here.
again I'm using a list of alive domains that contain about 6000~ domains.
and I'm running the following command: 
```cat domains.txt | ./nuclei -t nuclei-templates/vulnerabilities/ -o results.txt```
and after less than 2 seconds I get the following output and then exits...
```
[WRN] Use with caution. You are responsible for your actions
[WRN] Developers assume no liability and are not responsible for any misuse or damage.
[INF] [pdf-signer-ssti-to-rce] Loaded template PDF Signer v3.0 - SSTI to RCE via CSRF Cookie (@madrobot) [high]
[INF] [cached-pages-aem] Loaded template Invalidate / Flush Cached Pages From AEM (@hetroublemakr) [low]
[INF] [moodle-filter-jmol-xss] Loaded template Moodle filter_jmol XSS (@madrobot) [medium]
[INF] [Discourse XSS] Loaded template Discourse CMS XSS (@madrobot) [medium]
[INF] [crlf-injection] Loaded template CRLF injection (@nadino) [low]
[INF] [open-redirect] Loaded template Open Redirect Detection (@Elmahdi & @pxmme1337 & @Regala_ & @andirrahmani1) [low]
[INF] [wordpress-duplicator-path-traversal] Loaded template WordPress duplicator Path Traversal (@madrobot) [high]
[INF] [host-header-injection] Loaded template Host Header Injection (x-forwarded-host) (@melbadry9) [low]
[INF] [moodle-filter-jmol-lfi] Loaded template Moodle filter_jmol LFI (@madrobot) [high]
[INF] [wordpress-wordfence-xss] Loaded template WordPress Wordfence 7.4.6 Cross Site Scripting (@madrobot) [medium]
[INF] [twig-php-ssti] Loaded template Twig PHP <2.4.4 template engine - SSTI (@madrobot) [high]
```

**Comment by bauthard** [comment]
Hey, yes! 

for `HTTP` related templates you need URLs as input, for `dns` related templates you need to feed domains/subdomains, additionally, you can connect to us at [discord](https://discord.com/invite/KanAWHN) for more in general discussion and questions.

**Comment by Ice3man543** [comment]
Fixed by #108.
